content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -231,6 +235,11 @@ Update a FamilyMemberHistory.
 
     PUT /FamilyMemberHistory/:id
 
+_Implementation Notes_
+
+* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error'. Only existing conditions can be removed; condition id must be provided.
+* An update should only be preformed directly after a FamilyMemberHistory is retrieved. If a condition.id is provided from the get operation, it MUST be provided on the subsequent update and must match the condition.id returned from the get.
```

Comment:
for the get, let's say

condition.id is returned on the get operation.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/732/commits/247bf907d4433dd23adfb5dd1438bd3ae0bd78a6#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68R241